tests/contracts/test_policy_manager.py

```diff
@@ -264,4 +264,4 @@ def test_refund(web3, chain, escrow, policy_manager):
         period += 1
         tx = escrow.transact({'from': node}).mint(period)
         chain.wait.for_receipt(tx)
-    assert 140 == policy_manager.call().nodes(node)
+    assert 140 == policy_manager.call().nodes(node)
```

Comment:
Look at you with the int up front.  You liking that now?

Reply:
No, it's Yoda speak.